src/WPConfigTransformer.php

```diff
@@ -70,7 +70,7 @@ public function exists( $type, $name ) {
 			throw new Exception( 'Config file is empty.' );
 		}
 		// Normalize the newline to prevent an issue coming from OSX.
-		$this->wp_config_src = str_replace( array( "\n\r", "\r" ), "\n", $wp_config_src );
+		$this->wp_config_src = str_replace( array( "\r\n", "\n\r", "\r" ), "\n", $wp_config_src );
```

Comment:
As noted in the conversation, this change corrects the MS-DOS outlier by breaking the more common case.